tests/io/dask/dataframe/test_read.py

```diff
@@ -161,8 +161,10 @@ def test_reconstruct_dask_index(store_factory, index_type, monkeypatch):
     assert len(ddf.divisions) == 5
     assert ddf.divisions == (1, 2, 3, 4, 4)
 
-    assert_dask_eq(ddf_expected, ddf)
-    assert_dask_eq(ddf_expected_simple, ddf, check_divisions=False)
+    assert_dask_eq(ddf_expected, ddf, scheduler="distributed")
+    assert_dask_eq(
+        ddf_expected_simple, ddf, check_divisions=False, scheduler="distributed"
+    )
```

Comment:
We're secretly setting up distributed here https://github.com/data-engineering-collective/plateau/blob/433bf0730d620fa0447df9736a215051583fc06d/tests/io/dask/conftest.py#L14-L21

but `assert_dask_eq` is using `scheduler='sync'` as default. This is currently a conflict since P2P (picked by default since pyarrow is available and distributed is arround) is not implemented on sync